WebAutopark/Interfaces/IRepository.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace WebAutopark.Interfaces
+{
+    public interface IRepository<T> where T : class
+    {
+        void Create(T instance);
+        void Delete(int id);
+        T Get(int id);
+        List<T> GetAll();
+        void Update(T instance);
+    }
```

Comment:
Use async version of IRepository.
void -> Task
T -> Task<T>
Also use IEnumerble instead of any specific collection(like List). 
```suggestion
    public interface IRepository<T> where T : class
    {
        Task Create(T instance);
        Task Delete(int id);
        Task<T> Get(int id);
        Task<IEnumerable<T>> GetAll();
        Task Update(T instance);
    }
```

Reply:
Also Get is bad name.  Usually this method has GetById name

Reply:
![image](https://user-images.githubusercontent.com/79978383/118636052-eeadfc80-b7dc-11eb-8183-f7ee2c01aaba.png)
Done